src/annotator/sidebar.tsx

```diff
@@ -164,6 +164,22 @@ export class Sidebar implements Destroyable {
     this.bucketBar = null;
     this.features = new FeatureFlags();
 
+    // Set up the toolbar on the left edge of the sidebar.
+    const toolbarContainer = document.createElement('div');
+    this.toolbar = new ToolbarController(toolbarContainer, {
+      sidebarContainerId: this.iframeContainer?.id,
+      createAnnotation: (tool: AnnotationTool) => {
+        if (this._guestRPC.length === 0) {
+          return;
+        }
+
+        const rpc = this._guestWithSelection ?? this._guestRPC[0];
+        rpc.call('createAnnotation', { tool });
+      },
+      setSidebarOpen: open => (open ? this.open() : this.close()),
+      setHighlightsVisible: show => this.setHighlightsVisible(show),
+    });
```

Comment:
The whole block was lower, but moved it here so that we can append the toolbarContainer to the sidebarEdge element, and make the sidebarEdge the only absolute-positioned one, containing both the toolbar and the buckets right after.